flight/Modules/Stabilization/simulated/simsensors.c

```diff
@@ -570,21 +571,69 @@ static void simulateYasim()
 		last_baro_time = PIOS_DELAY_GetRaw();
 	}
 
+	// Update GPS periodically
+	static uint32_t last_gps_time = 0;
+	static float gps_vel_drift[3] = {0,0,0};
+	if(PIOS_DELAY_DiffuS(last_gps_time) / 1.0e6 > GPS_PERIOD) {
+		// Use double precision here as simulating what GPS produces
+		double T[3];
+		T[0] = homeLocation.Altitude+6.378137E6f * DEG2RAD;
+		T[1] = cosf(homeLocation.Latitude / 10e6 * DEG2RAD)*(homeLocation.Altitude+6.378137E6) * DEG2RAD;
```

Comment:
Spaces around operators. What is 6.378137E6f ? And the second copy of it doesn't have single precision suffix.

Reply:
I assume it's the distance to the middle of the earth, from MSL.. Though this is just factored from other code (and I dind't change spacing from the existing code)

Reply:
It should be a define. Things like that shouldn't need to be looked up to read code. :/